flink-runtime/src/test/java/org/apache/flink/runtime/zookeeper/ZooKeeperStateHandleStoreTest.java

```diff
@@ -803,17 +901,19 @@ public void testGetAll() throws Exception {
 
         for (Tuple2<RetrievableStateHandle<TestingLongStateHandleHelper.LongStateHandle>, String>
                 val : store.getAllAndLock()) {
-            assertTrue(expected.remove(val.f0.retrieveState().getValue()));
+            assertThat(expected.remove(val.f0.retrieveState().getValue())).isTrue();
         }
-        assertEquals(0, expected.size());
+        assertThat(expected.size()).isZero();
```

Comment:
```suggestion
        assertThat(expected).isEmpty();
```
nit: That's a really nitty remark. Feel free to ignore it.